src/util/pointerSupport.js

```diff
@@ -0,0 +1,24 @@
+import { addOnceEventListener } from './helpers'
+
+/**
+ * @mixin
+ */
+export default {
+  mounted () {
+    const setupPointerSupport = (propName, eventName) => {
+      addOnceEventListener(window, eventName, () => {
+        this.$vuetify[propName] = true
+        const className = 'application--' + propName.replace(/([A-Z])/g, '-$1').toLowerCase()
+        document.querySelector('[data-app]').classList.add(className)
+      })
+    }
+
+    // Adds application--touch-support class
+    // after touchstart event is triggered
+    setupPointerSupport('touchSupport', 'touchstart')
+
+    // Add application--hover-support class
+    // after mouseover event is triggered
+    setupPointerSupport('hoverSupport', 'mouseover')
```

Comment:
Does this work? Wasn't there something about mobile devices triggering `mouseover` as well?

Reply:
There was: https://github.com/Modernizr/Modernizr/issues/869
But I like the comment with this 2 letters less in every line :P

Reply:
Moreover, this PR (and touch detection in general) might be usesell when device has both touch and hover support...

Reply:
lol fair enough. 
That issue is from a few years ago, so things might have changed since then, but this class should be documented as being inaccurate if that's still the case.